.babelrc

```diff
@@ -51,7 +51,7 @@
       {
         "root": "./src",
         "alias": {
-          "@@vet360": "./src/platform/user/profile/vap-svc",
+          "@@vap-svc": "./src/platform/user/profile/vap-svc",
```

Comment:
This is the only real change. Everything else is an update to existing references to this alias.